axis.go

```diff
@@ -280,12 +280,10 @@ func (a *verticalAxis) size() (w vg.Length) {
 // draw draws the axis along the left side of a draw.Canvas.
 func (a *verticalAxis) draw(c draw.Canvas) {
 	x := c.Min.X
+	a.Label.TextStyle.Rotation += math.Pi / 2.
```

Comment:
That requires that there be a note on the axis documentation that the origin angle is at right angles to everything else, but OK.